src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -372,6 +431,33 @@ private void createProject(String projectId) throws Exception {
     assertNull(operationCow.getOperation().getError());
   }
 
+  /**
+   * Creates a notebook instance for the {@link InstanceName}. Blocks until the instance is created
+   * successfully or fails
+   */
+  private void createNotebookInstance(InstanceName instanceName)
+      throws IOException, InterruptedException {
+    OperationCow<com.google.api.services.notebooks.v1.model.Operation> operation =
+        notebooksCow
+            .operations()
+            .operationCow(
+                notebooksCow.instances().create(instanceName, defaultInstance()).execute());
+    operation =
+        OperationUtils.pollUntilComplete(operation, Duration.ofSeconds(30), Duration.ofMinutes(12));
```

Comment:
wow, supprise to see this may take upto 12 minutes.

Reply:
I haven't seen it take 10 minutes, but didn't want any flakes.